addons/vscode/src/extension.ts

```diff
@@ -83,7 +86,7 @@ export async function getCliPath(extensionPath?: string): Promise<string> {
 	state.configPath = configPath;
 
 	const checkExecutable = (path: string): string | null => {
-		const child = spawnSync(path, ['-V'], { timeout: 1000, encoding: 'utf8' });
+		const child = spawnSync(path, ['-V'], { timeout: PROBE_TIMEOUT, encoding: 'utf8' });
```

Comment:
if my guess in #259 is true, we can add another option `cwd` when actually launching the binary. And set cwd as project root